src/main/java/uk/gov/companieshouse/officer/delta/processor/processor/DeltaProcessor.java

```diff
@@ -127,14 +127,13 @@ private void handleResponse(final ResponseStatusException ex, final HttpStatus h
             throws NonRetryableErrorException, RetryableErrorException {
         Map<String, Object> logMap = DataMapHolder.getLogMap();
         logMap.put("status", httpStatus.toString());
-        if (HttpStatus.BAD_REQUEST == httpStatus) {
-            // 400 BAD REQUEST status is not retryable
+        if (HttpStatus.BAD_REQUEST == httpStatus || HttpStatus.CONFLICT == httpStatus) {
             logger.errorContext(logContext, msg, ex, logMap);
             throw new NonRetryableErrorException(msg, ex);
         }
         else if (httpStatus.is4xxClientError() || httpStatus.is5xxServerError()) {
```

Comment:
I think we should get rid of the else block below and make this the else block instead. We want to retry everything except 400 and 409.

Reply:
addressed in latest commit